spoon-runner/src/main/resources/page/log.html

```diff
@@ -24,6 +24,8 @@
                         <thead>
                             <tr>
                                 <th>Timestamp</th>
+                                <th>PID</th>
```

Comment:
We should figure out a way to only show these columns if the `showAllProcessesLog` boolean is true.